### PR TITLE
No clarion

### PR DESCRIPTION
**Pull request by kavanagh**

### CODE REVIEW

**Comment by kaelig** [comment]
These variables at the end of the file are deprecated, are you sure we should update them (and create new ones as well)?

**Comment by kavanagh** [comment]
ATM, if you unsilence o-fonts you get a bunch of warnings about Clarion. That's because o-fonts is itself using something that's deprecated. Meaning that even if no modules were using the deprecated vars you still get warnings.. This change prevents the warnings without removing the deprecated vars.

File: src/scss/_functions.scss [hunk 1]
```diff
@@ -47,7 +47,7 @@
 @function oFontsGetFontFamilyWithFallbacks($family) {
 	@if $family == 'Clarion' {
 		@warn "Clarion has been removed, use Georgia instead";
-		@return 'Georgia';
+		@return $o-fonts-serif;
```
Comment (kaelig) [inline src/scss/_functions.scss]: A mixin cannot `@return` anything, not sure why this is here in the first place. This should not output anything.

Comment (kavanagh) [inline src/scss/_functions.scss]: It's a @function.

Comment (kaelig) [inline src/scss/_functions.scss]: Oooh I was misreading completely. Sorry about that

**Pull request merged**